src/cockpit/asyncstdio.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/python3
```

Comment:
Is this file meant to be run by itself?

Reply:
No.  It is not.

Reply:
Then let's remove the `#!` line, I'd say.